### PR TITLE
[RDS-1303] - Refactor manifest load, use denomination

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: src/pages/Redstone/Feeds/components/FeedsTable.vue [hunk 1]
```diff
@@ -104,7 +104,16 @@
               token: toUrlParam(item.token),
             },
           }"
-          >{{ clearFeedName(item.feed) }}</RouterLink
+        >
+          <span v-if="item.token.includes('/')">{{ clearFeedName(item.token) }}</span>
```
Comment (cehali) [inline src/pages/Redstone/Feeds/components/FeedsTable.vue:108]: we still have tokens like `SolvBTC_MERLIN/BTC-TWAP-60`. TWAP-60 is not denomination, it is information that whole token is calculated based on TWAP aggregation

File: src/store/manifests.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    async getOrderedProviders({ dispatch }) {
+      const manifests = await dispatch('initializeManifests');
+      return Object.keys(manifests);
+    },
+
+    setLoadComplete({ commit }) {
+      commit('SET_LOAD_COMPLETE', true);
+    }
+  }
+};
```
Comment (cehali) [inline src/store/manifests.js:201]: new line

File: src/store/manifests.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        tokenDetails[symbol] = { 
+          ...config, 
+          providers: [DEFAULT_PROVIDER],
+          config: {} // Initialize empty config for all tokens
+        };
+      });
+    
+      // Process each manifest and collect all configs
+      for (const [provider, manifest] of Object.entries(manifestsObject)) {
+        for (const [symbol, config] of Object.entries(manifest.tokens || {})) {
```
Comment (cehali) [inline src/store/manifests.js:144]: WDYT about optimizing this part a little bit? Manifest are rather big and have a lot of repetitions

Comment (roszpun) [reply src/store/manifests.js:144]: sounds good!

File: src/pages/Redstone/Feeds/components/FeedsTable.vue (hunk 1 above)
Comment (roszpun) [reply src/pages/Redstone/Feeds/components/FeedsTable.vue:108]: It will be SolvBTC_MERLIN/BTC with TWAP-60 icon.

Comment (roszpun) [reply src/pages/Redstone/Feeds/components/FeedsTable.vue:108]: Screenshot 2024-11-26 at 11 58 48

**Review by cehali — APPROVED** [review]

**Review by JakubJaksik — APPROVED** [review]

**Pull request merged**